src/js/components/meta/FormiflyContext.tsx

```diff
@@ -270,12 +328,15 @@ export const FormiflyProvider = (props) => {
                         setSubmitting(false);
                         setSubmitSuccess(true);
                     }
+                    return;
                 } catch (err) {
                     setSubmitFailureReason(err);
+                    return;
                 }
             })
-            .catch((reason) => {
-                let newErrors = {};
+            .catch((reason: UnpackedErrors<T>) => {
+                // todo: isn't this just copying reason to newErrors? why?
```

Comment:
the keys within reason are written as `object.key` and the object has a flat structure. The value for Errors has to be a non-flat structure, so we need to copy the values like this

Reply:
That is not true. On a reject, unpackErrors is called and the result is returned as the rejection value. unpackErrors already generates a non-flat object, thus we are only copying here. I just validated this by looking at the code but also by testing. Did I misunderstand something?